### PR TITLE
Bug fix added validation Issue #31

### PR DESCRIPTION
**Pull request by sadath2001**
Hi,

Regarding issue #31 
I have added validation for checking if the ingredients and steps are populated fore adding next steps and ingredients.
Also added nodemon instead of node as its more efficient while testing out.

Kindly add this under gssoc ext and hacktober

### CODE REVIEW

**Comment by sadath2001** [comment]
Hi @AlfiyaSiddique , kindly let me know if any changes needed.
also kindly add gssoc extended and hacktober fest in this PR.
thanks.

File: backend/package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js",
```
Comment (AlfiyaSiddique) [inline backend/package.json:9]: This affects the deployment, 
Revert starts as it was i.e node server.js
`dev` can be used to run with nodemon but revert the `start` command

File: frontend/common/backendUrl.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// const backendURL  = 'http://localhost:3000' // local environment
+// const backendURL  = 'http://localhost:4000' // local environment
```
Comment (AlfiyaSiddique) [inline frontend/common/backendUrl.js:1]: Revert this as well as it was before.
Also, note to revert all the config changes back to as it was before pushing to the PR.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "install": "^0.13.0"
+  }
+}
```
Comment (AlfiyaSiddique) [inline package.json:5]: Why did you add an `install` dependency in the outer part of frontend and backend folder?
I think it is by mistake, as I can't see this dependency being used anywhere please remove it.
Both this file and package-lock.json which is outside frontend and backend folder needs to be removed.

**Comment by AlfiyaSiddique** [comment]
@sadath2001 Some Comments.
Please fix them all

**Review by sadath2001 — COMMENTED** [review]
Changed the required things as requested

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**